views/welcome.ejs

```diff
@@ -34,5 +34,16 @@
 
     <p>We collect anonymous analytics using Simple Analytics. View live: <a href="#">simpleanalytics.com/learndatabases.dev</a></p>
 
+    <script>
+
+      fetch('/api/createDatabase', {
```

Comment:
How is this supposed to work?
Is the landing page gonna automatically crate database when the user connects?